go/vt/schema/parser.go

```diff
@@ -109,7 +109,7 @@ func legacyParseRevertUUID(sql string) (uuid string, err error) {
 // ParseEnumValues parses the comma delimited part of an enum column definition
 func ParseEnumValues(enumColumnType string) string {
 	if submatch := enumValuesRegexp.FindStringSubmatch(enumColumnType); len(submatch) > 0 {
-		return submatch[1]
+		return submatch[2]
```

Comment:
Is this fixing a bug?

Reply:
This isn't fixing a bu. This is because the regex changed from `enum` to `(enum|set)`, which adds a new `(...)` submatch to the pattern. There are unit tests to validate the correctness of the submatch in this function.